### PR TITLE
ric-16: adding rp3 env setup script

### PR DESCRIPTION
**Pull request by sahellebusch**
Added script to setup RP3 env to accept and run ric.

### CODE REVIEW

File: scripts/rp3-setup.sh [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  rm -rf ~/.npm
+  rm -rf node_modules/
+}
+
+echo "Checking node version..."
+if [ $(command -v node) ]
+then
+  if [[ $(node --version) == *"v8."* ]]
+  then
+    echo "Node version 9 is already installed!  Moving on."
```
Comment (johnfoley3) [inline scripts/rp3-setup.sh]: Should be "Node version 8"

How easy is bash interpolation? the setup link above is standard across recent node major version install scripts, so if we wanted to be fancy, we could set a var above to what version of Node we want, and then interpolate it into areas we want to check for.

assuming `${ string }`:
` curl -sL https://deb.nodesource.com/setup_${NODE_VERSION}.x | sudo -E bash - `
` if [[ $(node --version) == *"v${NODE_VERSION}."* ]]`
  etc

File: scripts/rp3-setup.sh [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+else
+ install_node
+fi
+
+if [ $? -ne 0 ]
+then
+  echo -e "\e[91mFailed to install or update NodeJS."
+fi
+
+echo "Removing old docker version..."
```
Comment (johnfoley3) [inline scripts/rp3-setup.sh]: I think there is a missing "uninstall docker" command here. `echo` will be successful most of the time, as long as a `tty` was available to print on, so the next if statement will pass most of the time.

File: scripts/rp3-setup.sh [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+if [ $? -ne 0 ]
+then
+  echo "Failed to retrieve latest version:" >&2
+  exit $$
+fi
+
+echo  "Installing docker..."
+sh get-docker.sh 2> /dev/null
+$(rm get-docker.sh)
```
Comment (johnfoley3) [inline scripts/rp3-setup.sh]: What is the difference between running a command in or out of `$(...)`?

File: scripts/rp3-setup.sh (hunk 2 above)
Comment (johnfoley3) [reply scripts/rp3-setup.sh]: BTW I consider uninstalling docker to be a good-to-have, so no need to fix it now if figuring out how to uninstall it is a pain

**Comment by sahellebusch** [comment]
Since this is all going to live in the new `ric` module, I'll address comments in the next PR there.

**Pull request closed without merging**